.github/workflows/release-dotnet-native-spanner-lib.yml

```diff
@@ -0,0 +1,114 @@
+on:
+  workflow_dispatch:
```

Comment:
This means that these scripts can only be executed manually through the GitHub API. They do not run for any pull requests or anything like that.